itou/templates/search/prescribers_search_results.html

```diff
@@ -10,78 +10,92 @@
-                                    <div class="card my-4">
-                                        <div class="card-body">
-                                            <h3 class="h2 card-title">
-                                                <a href="{{ prescriber_org.get_card_url }}?back_url={{ request.get_full_path|urlencode }}">
-                                                    {{ prescriber_org.name }}
-                                                </a>
-                                            </h3>
-                                            <h4 class="h5 card-subtitle mb-2 text-muted">
-                                                {{ prescriber_org.get_kind_display }}
-                                                {% if prescriber_org.is_brsa %}(conventionné par le Département pour le suivi des BRSA){% endif %}
-                                            </h4>
-                                            <hr>
-                                            <h4 class="h5 card-subtitle mb-2 text-muted">{{ prescriber_org.address_on_one_line }}</h4>
-                                            <p class="small mb-0 card-text">
-                                                <span class="badge badge-pill badge-info">{{ prescriber_org.distance.km|floatformat:"-1" }} km</span>
-                                                de votre lieu de recherche
-                                            </p>
-                                        </div>
-                                    </div>
-                                {% empty %}
-                                    <div class="card my-4">
-                                        <div class="card-body">
-                                            <p class="font-weight-normal text-muted">Aucun résultat avec les filtres actuels.</p>
-                                        </div>
-                                    </div>
-                                {% endfor %}
-
-                                {% include "includes/pagination.html" with page=prescriber_orgs_page %}
-                            </div>
+        <section class="s-box mt-3">
+            <div class="s-box__container container">
+                <div class="s-box__row row">
+                    <div class="col-12 col-md-4">{% include "search/includes/prescribers_search_filters.html" with form=form %}</div>
+                    <div class="col-12 col-md-8">
+                        <div class="mb-3 mb-md-4">
+                            <h3 class="h4 mb-0">
+                                {% with prescriber_orgs_page.number as current_page and prescriber_orgs_page.paginator.num_pages as total_pages and prescriber_orgs_page.paginator.count as counter %}
+                                    {{ counter }} résultat{{ counter|pluralize }}
+                                    {% if prescriber_orgs_page.paginator.num_pages > 1 %}- Page <b>{{ current_page }}</b>/{{ total_pages }}{% endif %}
```

Comment:
Autant utiliser la variable :)
```suggestion
                                    {% if total_pages > 1 %}- Page <b>{{ current_page }}</b>/{{ total_pages }}{% endif %}
```